rbx_binary/src/tests/serializer.rs

```diff
@@ -124,3 +124,30 @@ fn logical_properties_basepart_size() {
     let decoded = DecodedModel::from_reader(buffer.as_slice());
     insta::assert_yaml_snapshot!(decoded);
 }
+
+/// Ensures that Color3 can be serialized as Color3uint8. We use both the
+/// canonical property name Part.Color as well as the serialized name
+/// Part.Color3uint8.
+#[test]
+fn color3_to_color3uint8() {
+    let tree = WeakDom::new(
+        InstanceBuilder::new("Folder")
+            .with_child(
+                InstanceBuilder::new("Part")
+                    .with_property("Color3uint8", Color3::new(-0.25, 0.5, 1.2)),
+            )
+            .with_child(
+                InstanceBuilder::new("Part")
+                    .with_property("Color", Color3::new(0.1111, 0.3333, 0.9999)),
```

Comment:
Can we change this property to be `Color3uint8` with a `Color3uint8` value? As is, this is redundant with the third instance. Testing all of the valid combinations here is a good idea I think.